sources/niUI/src/GDRV_GLES2.cpp

```diff
@@ -36,15 +36,15 @@ static GLint samplerFilterAnisotropySharp = 8;
 //  Cache
 //
 //--------------------------------------------------------------------------------------------
-enum eGLCache
-{
+enum eGLCache {
```

Comment:
Please try to avoid committing "formatting only" changes, it makes reviewing harder. Eventually we'll add a proper .clang-format and mandate all the code to go through it.